35-CAN-vehicle-monitor/firmware/README.md

```diff
@@ -0,0 +1,147 @@
+
+* [Visual Studio Code (VSCode)](https://code.visualstudio.com/) with the [PlatformIO extension](https://platformio.org/)
+* USB C cable
+* [USB to CAN Converter Cable](https://www.amazon.com/gp/product/B09K3LL93Q)
+* [Adafruit Feather M4 CAN Express with ATSAME51](https://www.adafruit.com/product/4759)
+* [Notecarrier-F](https://shop.blues.io/products/notecarrier-f)
+* [Notecard](https://blues.io/products/notecard/)
+* Antenna (comes with the Notecard)
+* 2 male-to-male jumper wires
+* Soldering iron
+* Solder flux
+
+## Hardware Setup
+
+### MCU
+
+The Adafruit Feather M4 CAN Express (the "MCU") will receive CAN traffic from your development PC, filter that traffic based on the IDs in the messages, and pass the filtered messages on to the Notecard for transmission to Notehub. The MCU comes with male headers and a CAN transceiver screw terminal block (the "terminal block") that need to be soldered onto the MCU board in order for it to plug into the Notecarrier F.
+
+![MCU and components, not soldered](../images/mcu_not_soldered.jpg)
+
+Adafruit has a [handy guide for soldering headers](https://learn.adafruit.com/how-to-solder-headers/overview). Once you've soldered on the headers and the terminal block, your MCU should look like this:
+
+![MCU and components, soldered](../images/mcu_soldered.jpg)
+
+Now, plug the headers of the MCU into the female headers on the Notecarrier F. Install the Notecard into the Notecarrier and the antenna onto the Notecard as described in the [Notecard Quickstart guide](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-f/#connect-your-notecard-and-notecarrier). Finally, connect the MCU to your development PC with a USB C cable.
+
+### USB to CAN Converter
+
+The USB to CAN converter (the "converter") allows us to send CAN messages from our development PC to the MCU via a CAN bus. A CAN bus requires two signal lines: CAN high (CANH) and CAN low (CANL). The converter comes in two parts. Plug the part with the USB connector into your development PC. Connect the second part to the first part via the RS232 connector. This second part has a screw terminal block with ports labeled CANH and CANL. Using a Philips head screwdriver, loosen the screws above these two connectors, insert the end of a male-to-male jumper wire into each, and tighten the screws, securing the wires. Similarly, take the free ends of each jumper wire and plug the CANL wire into the port labeled L on the MCU's terminal block and the CANH wire into the port labeled H. At this point, the hardware is fully set up and should look like this:
+
+![Complete hardware](../images/complete_hardware.jpg)
+
+## Notehub Setup
+
+Sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-pi/#set-up-notehub).
+
+## Firmware Setup
+
+To build and upload the firmware onto the MCU, you'll need VSCode with the PlatformIO extension.
```

Comment:
Very trivial, but technically “VS Code” is correct so you should probably do a quick replace all in here.